examples/http/src/main/java/io/opentelemetry/example/http/HttpServer.java

```diff
@@ -55,25 +55,26 @@ public void handle(HttpExchange he) throws IOException {
       try (Scope scope = ContextUtils.withScopedContext(ctx)) {
         // Build a span automatically using the received context
         span = spanBuilder.startSpan();
-      }
 
-      // Set the Semantic Convention
-      span.setAttribute("component", "http");
-      span.setAttribute("http.method", "GET");
-      /*
-      One of the following is required:
-      - http.scheme, http.host, http.target
-      - http.scheme, http.server_name, net.host.port, http.target
-      - http.scheme, net.host.name, net.host.port, http.target
-      - http.url
-      */
-      span.setAttribute("http.scheme", "http");
-      span.setAttribute("http.host", "localhost:" + HttpServer.port);
-      span.setAttribute("http.target", "/");
-      // Process the request
-      answer(he, span);
-      // Close the span
-      span.end();
+        // Set the Semantic Convention
+        span.setAttribute("component", "http");
+        span.setAttribute("http.method", "GET");
+        /*
+         One of the following is required:
+         - http.scheme, http.host, http.target
+         - http.scheme, http.server_name, net.host.port, http.target
+         - http.scheme, net.host.name, net.host.port, http.target
+         - http.url
+        */
+        span.setAttribute("http.scheme", "http");
+        span.setAttribute("http.host", "localhost:" + HttpServer.port);
+        span.setAttribute("http.target", "/");
+        // Process the request
+        answer(he, span);
```

Comment:
(This `heh` golang style param name distracts me, hah)

Reply:
fixed!